index.html

```diff
@@ -1,12 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./scss/main.css" />
```

Comment:
The Styling is not valid. You need to change from `./scss/main.scss` to `./css/main.scss`.
Then I could see your styling correctly:D
```suggestion
    <link rel="stylesheet" href="./css/main.css" />
```

Reply:
thanks for your review! I fixed!